build-system/tasks/extension-helpers.js

```diff
@@ -776,6 +768,42 @@ async function generateBentoEntryPointSource(name, options) {
     .replace('__name__', JSON.stringify(name));
 }
 
+/**
+ * @param {string} dir
+ * @param {string} name
+ * @param {Object} options
+ * @return {Promise<string>}
+ */
+async function getBentoWebComponentFilename(dir, name, options) {
```

Comment:
can we merge/replace this with the `getBentoFilename()`? Looks like they do the same thing

Reply:
okay sounds good! The differences i noticed were
1. the other fn conditionally imports CSS, which I think we'll need here too.
2. the other fn saves to `build/bento-foo.js` and makes a new file for each ext and this fn saves to `build/web-component.js`. i like your way since it creates fewer files unnecessarily, but the other way could be better if we need them.

lmk if i missed something

Reply:
ok combined - ptal :)